src/main/java/com/suse/saltstack/netapi/client/SaltStackClient.java

```diff
@@ -457,4 +457,30 @@ public Keys call() throws SaltStackException {
     public EventStream events() {
         return new EventStream(config);
     }
+
+    /**
+     *
+     * @param eventTag
+     * @param eventData
+     * @return
+     * @throws SaltStackException
```

Comment:
Please complete the javadoc: put a description of the method, parameters, return value and when an exception is thrown.